File: design-patterns/java/java-observer-design-pattern/Group.java
public class Group implements Follower {
    private String name;

    public Group(String name) {
        this.name = name;
    }

    @Override
    public void notify(Post post) {
        System.out.println(this.name + " => the user/group you are following (" + post.toString() + ") has shared: " + post.getContent());
    }
}

File: design-patterns/csharp/csharp-observer-design-pattern/Group.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace csharp_observer_design_pattern
{
    public class Group : Follower
    {
        private string name;

        public Group(string name)
        {
            this.name = name;
        }

        public void Notify(Post post)
        {
            Console.WriteLine($"{this.name} => the user/page you are following ({post.ToString()}) has shared: {post.Content}");
        }    
    }
}